### PR TITLE
Adds/updates the payload commands to the reference guide

### LINKED ISSUE #927: No mention of storage or payloads on metadata.yaml page
Storage and payloads are both available in 1.25, but neither are mentioned here: https://jujucharms.com/docs/1.25/authors-charm-metadata

There is a page devoted to storage here: https://jujucharms.com/docs/1.25/storage. Perhaps we could link to that.

There is some info about payloads in the 1.25.0 release notes here: https://jujucharms.com/docs/1.25/reference-release-notes

### PR DESCRIPTION
**Pull request by chuckbutler**
fixes #927

### CODE REVIEW

File: src/en/reference-hook-tools.md [hunk 1]
```diff
@@ -296,6 +296,28 @@ open-port 80/tcp
 open-port 1234/udp
 ```
 
+## payload-status-set
+
+`payload-status-set` is used to update the current status of a registered payload.
+The `class` and `id` provided must match a payload that
+has been previously registered with juju using [payload-register](#payload-register). The <status> must be one of the following:
```
Comment (mbruzek) [inline src/en/reference-hook-tools.md]: You can not use less-than and greater-than symbols in markdown like that. When rendered I do not see "&lt;status&gt;"

Comment (evilnick) [inline src/en/reference-hook-tools.md]: I would be tempted to leave them out - we aren't referring to an example or anything that we have identified as '&lt;status>'

Comment (evilnick) [inline src/en/reference-hook-tools.md]: otherwise you need to use `&lt;`

Comment (mbruzek) [inline src/en/reference-hook-tools.md]: `&lt;` and `&gt;` is how I got the status to render in the message.

**Comment by chuckbutler** [comment]
Pushed a fix on that last commit. LMK if there's anything else in here that needs doing.

**Comment by evilnick** [comment]
:cake:

**Comment by evilnick** [comment]
P.S., do you have examples for network-get too? #1200

**Pull request merged**